pkg/validations/cmdvalidations/cmdvalidations.go

```diff
@@ -0,0 +1,124 @@
+package cmdvalidations
```

Comment:
What is this package supposed to contain? I mix match of different things that I'm not sure belong in the same place.

Reply:
I'm packaging the validations into validation results and grouping some individual checks into single validation. Should grouping the validations be moved out to the broader validations package?